mage_ai/data_preparation/storage/local_storage.py

```diff
@@ -39,22 +39,36 @@ def remove(self, path: str) -> None:
     def remove_dir(self, path: str) -> None:
         shutil.rmtree(path, ignore_errors=True)
 
-    def read_json_file(self, file_path: str, default_value: Dict = None) -> Dict:
+    def read_json_file(
```

Comment:
Could you also update S3Storage and GCSStorage? Otherwise, it'll throw exceptions when using those two storage classes.

Reply:
I made it so that the behavior doesn't change for those classes. The default value of `raise_exception` is `False`, so the exceptions shouldn't be thrown in those cases.

Reply:
Have you tested it? 
The S3Storage class doesn't support `raise_exception` param in the `read_json_file` method right now. If you call it, I suppose it'll throw exception: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/storage/s3_storage.py#L53

Reply:
oh sorry, i thought you were talking about something else. not sure i missed those, i'll update them